source/_includes/integrations/config_flow.md

```diff
@@ -3,7 +3,7 @@
 
 ## Configuration
 
-To add the **{{ name }}** integration to your Home Assistant instance, use this My button:
+To add the **{{ name }}** {% unless page.ha_integration_type == 'hardware' or page.ha_integration_type == 'virtual' %}{{ page.ha_integration_type | default: "integration" }}{% endunless %} to your Home Assistant instance, use this My button:
```

Comment:
```suggestion
To add the **{{ name }}** {% unless page.ha_integration_type == 'hardware' or page.ha_integration_type == 'virtual' %}{{ page.ha_integration_type | default: "integration" }}{% else %}integration{% endunless %} to your Home Assistant instance, use this My button:
```
Maybe add an else statement to preserve the "integration", if nothing is shown otherwise?